src/KiD.jl

```diff
@@ -13,17 +13,17 @@ import Thermodynamics
 import CloudMicrophysics
 import CLIMAParameters
 
-using OrdinaryDiffEq: ODEProblem, solve, SSPRK33
+using OrdinaryDiffEq: ODEProblem, solve, SSPRK33, Tsit5
 
 import Logging
 import TerminalLoggers
 Logging.global_logger(TerminalLoggers.TerminalLogger())
 
+const TD = Thermodynamics
 const CP = CLIMAParameters
-const APS = CP.AbstractEarthParameterSet
 
```

Comment:
Can we also add: `const FT = Float64` or something like that?

Reply:
I think I fixed it. We specify `FT` once in the simulation driver file and it should now work. 
The problem was twofold 
a) I was trying to pass `FT` to the `drho_dt` function for ODEsolver (which I can't unless I hide it in `p`) 
b) Revise was failing to revise changes to that function.